### PR TITLE
V0.19.16

### PR DESCRIPTION
**Pull request by kirk-patton**
Update k8s.io/*@v0.19.16

reference: https://westcoastdevops.slack.com/archives/C01F5CZU6LU/p1652906498384179

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,38 +3,25 @@ module k8s.restdev.com/operators
 go 1.15
```
Comment (scr-oath) [inline go.mod]: Can this bump to at least 1.16? I know we've all been using 1.18 but 1.16 is the minimum usable on M1 macs (go for ARM chip wasn't avail before then)

File: hack/update-codegen.sh [hunk 2]
```diff
@@ -22,24 +22,25 @@ RED='\033[0;31m'
 GREEN='\033[0;32m'
 NC='\033[0m'
 
-SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
-CODEGEN_PKG="../../k8s.io/code-generator"
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
```
Comment (scr-oath) [inline hack/update-codegen.sh:25]: I'm wondering if this is something that could use the `tools.go` style of doing things, like we added for `mockgen`…

https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module

File: go.mod (hunk 1 above)
Comment (ericuldall) [reply go.mod]: we can bump it to any version we like methinks. It runs in docker so we just need to update the docker file accordingly

File: hack/update-codegen.sh (hunk 2 above)
Comment (ericuldall) [reply hack/update-codegen.sh:25]: we do have to tools.go file but i'm not sure what else is needed to make this look the way you are expecting. checking the resource you sent

Comment (ericuldall) [reply hack/update-codegen.sh:25]: see https://github.com/West-Coast-Devops/scheduled-scaler/pull/55/files#diff-c70c0ccc1e4ba2d6da1d6c59877e3600c6771f133dec670df485e23fde6cb795

File: go.mod (hunk 1 above)
Comment (kirk-patton) [reply go.mod]: updated to 1.17

**Review by ericuldall — APPROVED** [review]

**Pull request merged**